controller/src/main/resources/reference.conf

```diff
@@ -10,7 +10,7 @@
 
 CONTROLLER_SERVER_PORT = 9090
 
-ASYNC_TASK_POOL_SIZE = 20
+ASYNC_TASK_POOL_SIZE = 80
```

Comment:
To goal of this PR is to consolidate multiple thread pools into a single thread pool, and not optimize the cumulative number of threads created across all thread pools. Earlier, we were creating 4 different thread pools each of `ASYNC_TASK_POOL_SIZE` size. Since we have now consolidated them into a single thread pool, modified the default value of `ASYNC_TASK_POOL_SIZE` to 4 times original value.